static/css/cf-enhancements.less

```diff
@@ -804,3 +807,38 @@ tbody {
     name: EOF
     eof: true
 */
+
+
+/* topdoc
+  name: Warning
+  family: cf-forms
+  notes:
+    - "Adds Warning state to form fields"
+  tags:
+    - cf-forms
+*/
+
+@input-warning:     @gold;
+
+input[type="text"],
```

Comment:
This is a direct copy from cf-forms. Opting to leave it so when it's ported back it matches the others.